federation/.env.dist

```diff
@@ -0,0 +1,17 @@
+LOG_LEVEL=debug
+PORT=4000
+GRAPHIQL=true
+
+# Database
+DB_PORT=3306
+DB_DATABASE=gradido_community
```

Comment:
where is the rest of the database configs?

Reply:
added